platform/android/os_android.cpp

```diff
@@ -448,6 +479,10 @@ String OS_Android::get_data_path() const {
 	return get_user_data_dir();
 }
 
+String OS_Android::get_executable_path() const {
+	return OS::get_executable_path();
```

Comment:
Is it intentional to bypass `OS_Unix::get_executable_path()` by going straight to `OS`?
If so, this warrants a commit to clarify (otherwise it looks like either a redundant override or a mistake).

Reply:
Yes, this is done so we can detect when we're running on Android, in which case `apk` is returned as the executable path, and act accordingly.
The unix logic returns the unix process path whose access can be limited depending on which version of Android the device is running.